Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -239,6 +248,8 @@ open class TextStorage: NSTextStorage {
         detectAttachmentRemoved(in: range)
         textStore.replaceCharacters(in: range, with: str)
 
+        replaceTextStoreString(range, with: str)
```

Comment:
We should move the update logic inside of `override func edited()` (inside the next call in this method), and only trigger it for `.editedCharacters`.

That way we don't need to call both at several point in the app.